packages/excalidraw/components/Actions.tsx

```diff
@@ -104,7 +104,7 @@ export const SelectedShapeActions = ({
   ) {
     isSingleElementBoundContainer = true;
   }
-  const isEditing = Boolean(appState.editingElement);
+  const isEditingText = Boolean(appState.editingTextElement);
```

Comment:
this is technically a regression as it excludes when creating arrows. I've added a check for `newElement` as well